pylint/lint/pylinter.py

```diff
@@ -282,13 +281,18 @@ def make_options() -> Options:
                 {
                     "default": "text",
                     "type": "string",
+                    # pylint: disable-next=fixme
+                    # TODO: Optparse: Uncomment these lines.
```

Comment:
Uncommenting this will fix the coverage drop in this PR but that requires completely fixing the `PyLinter` class for `argparse`. I have all PRs for that ready to go but I'm breaking them up to keep them reviewable.